src/main/java/org/kettingpowered/ketting/internal/KettingConstants.java

```diff
@@ -67,12 +67,7 @@ public class KettingConstants {
         BUKKIT_PACKAGE_VERSION = String.join("_", mcv);
         BUKKIT_VERSION = MINECRAFT_VERSION + "-R0.1-SNAPSHOT";
 
-        final String MC_FORGE_KETTING = MINECRAFT_VERSION + "-" + FORGE_VERSION + "-" + KETTING_VERSION;
-        final String TYPE_NAME = TYPE.typeOrThrow();
-        final String UNIVERSAL_NAME = TYPE_NAME + "-" + MC_FORGE_KETTING + "-universal.jar";
-        final File INSTALL_DIR = TYPE.installDirOrThrow();
```

Comment:
Removing variables is always tricky, since older ketting versions autoupdate, if we release a new version.
Also another thing to note, is my comment on the kettinglauncher pr.